lib/Filters/GitDiffFilter.cpp

```diff
@@ -0,0 +1,97 @@
+#include "mull/Filters/GitDiffFilter.h"
+
+#include "mull/Diagnostics/Diagnostics.h"
+#include "mull/Toolchain/Runner.h"
+
+#include "llvm/IR/InstIterator.h"
+#include <llvm/IR/DebugInfoMetadata.h>
+#include <llvm/IR/DebugLoc.h>
+#include <llvm/IR/Function.h>
+#include <llvm/Support/FileSystem.h>
+
+#include <iostream>
+#include <regex>
+#include <sstream>
+#include <unistd.h>
+
+using namespace mull;
+
+static std::string getCwd() {
```

Comment:
Do we already have this kind of utils somewhere?

Reply:
We better use this one [`llvm::sys::fs::current_directory`](https://llvm.org/doxygen/namespacellvm_1_1sys_1_1fs.html#aec50dfe28ae22b8ff45f02ca193b7bf7)